src/dfcx_scrapi/tools/test_cases_util.py

```diff
@@ -0,0 +1,552 @@
+"""Test Cases Utility functions."""
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Dict, List
+import time
+from google.cloud.dialogflowcx_v3beta1 import types
+
+from dfcx_scrapi.core import scrapi_base
+from dfcx_scrapi.core import flows
+from dfcx_scrapi.core import pages
+from dfcx_scrapi.core import intents
+from dfcx_scrapi.core import test_cases
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+class TestCasesUtil(scrapi_base.ScrapiBase):
+    """Util class for CX test cases"""
+
+    def __init__(
+        self,
+        creds_path: str = None,
+        creds_dict: Dict = None,
+        creds=None,
+        scope=False,
+        agent_id: str = None,
+        test_case_id: str = None,
+    ):
+        super().__init__(
+            creds_path=creds_path,
+            creds_dict=creds_dict,
+            creds=creds,
+            scope=scope,
+        )
+
+        if agent_id:
+            self.agent_id = agent_id
+            self.client_options = self._set_region(self.agent_id)
+
+        if test_case_id:
```

Comment:
Remove this section, none of these are used in this class.

Reply:
Removed